frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1655,6 +1655,12 @@ https://archiveweb.page/images/${"logo.svg"}`}
         </sl-checkbox>
       `)}
       ${this.renderHelpTextCol(infoTextFor["blockAds"], false)}
+      ${inputCol(html`
+        <sl-checkbox name="saveStorage" ?checked=${this.formState.saveStorage}>
+          ${msg("Save local and session storage")}
+        </sl-checkbox>
+      `)}
+      ${this.renderHelpTextCol(infoTextFor["saveStorage"])}
```

Comment:
This can be better documented, but passing `false` (like the info text for `blockAds`) will top-align the info text with the form control.
```suggestion
      ${this.renderHelpTextCol(infoTextFor["saveStorage"], false)}
```